server/server.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/allegro/bigcache"
+)
+
+const (
+	apiVersion  = "v1"
+	apiBasePath = "/api/" + apiVersion + "/"
+
+	// path to cache.
+	cachePath = apiBasePath + "cache/"
+)
+
+var (
+	shards             int
+	maxEntriesInWindow int
+	lifeWindow         int64
+	hardMax            int
+	verbose            bool
+	maxEntrySize       int
+	logfile            string
+	port               int
```

Comment:
Instead of declaring fields for every config entry we can just use config struct see https://github.com/allegro/marathon-consul/blob/1.4.2/config/config.go#L63